### DISCUSSION TITLE
Roll back to stock

### DISCUSSION THREAD

**Discussion by tbilic14** [Mr-MIBonk/M.I.B._More-Incredible-Bash#240] [Q&A]
I installed MHI2_ER_AU57×_K3663_1, is it safe to flash back to stock MHI2_ER_AU57×_P3634 and what should I make sure before doing it?

I read something about sound system might be affeceted and don’t quite understand what it means and how to make sure all goes ok…

**Comment by harman-f** [comment]
Why do you want to go back to MHI2_ER_AU57×_P3634?

MHI2_ER_AU57×_P3634 was installed originally on that unit?

**Reply by tbilic14** [reply]
Yes it was originally installed. I’m selling the car and I figure better leave it original.

**Reply by harman-f** [reply]
Leave it like it is. Going back to an older FW has a high potential of drama ;)

**Reply by tbilic14** [reply]
Thanks for the advice. Will do so. 
Just out of curiosity, can you describe one potential drama? :)

**Reply by harman-f** [reply]
You have to downgrade FW with modified metainfo or force train ignore via SWDL
Some coding has to be reverted, original FecContainer copied back.
